utils/cryptodev-linux/Makefile

```diff
@@ -10,12 +10,15 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=cryptodev-linux
-PKG_VERSION:=1.8
-PKG_RELEASE:=2
+PKG_VERSION:=2017-02-09
```

Comment:
It is a kernel module and thus can be upgraded. Current version seems good; merging.